src/components/squareComponent/square.spec.js

```diff
@@ -1,17 +1,90 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import GameSquare from './squareComponent';
-import Enzyme from 'enzyme';
+import Enzyme, { mount } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
 Enzyme.configure({ adapter: new Adapter() });
 
 describe('Square Component', () => {
-
   it('renders the Square component without crashing', () => {
     const gameSpaceDiv = document.createElement('div');
+
     ReactDOM.render(<GameSquare />, gameSpaceDiv);
     ReactDOM.unmountComponentAtNode(gameSpaceDiv);
+
+  });
+
+  describe('Mounting the square component', () => {
+    it('it is empty when initially mounted', () => {
+      const wrapper = mount(<GameSquare />);
+
+      expect(wrapper.state('X')).toBe('');
+      expect(wrapper.state('O')).toBe('');
+
+    });
+
+  });
+
+  describe('Clicking a square', () => {
+    it('handles placing an X on the square', () => {
+      const wrapper = mount(<GameSquare />);
+
+      expect(wrapper.state('X')).toBe('');
+
+      wrapper.find('.spaces').simulate('click');
+
+      expect(wrapper.state('X')).toBe('X');
+
+    });
+
+    it('handles placing an O on the square', () => {
```

Comment:
This test also has too many newlines. The Arrange/Act/Assert approach is meant to have exactly two newlines. One in-between the Arrange and the Act, and a second between the Act and the assert.

Reply:
I don't really understand the spacing AAA pattern with my test cases.  Because I have more than one assertion separated by an action it's causing me to have more than 2 lines. 

Perhaps I could remove the first assertion?